### ISSUE TITLE
Accessibility: The list item of comboBox incorrectly displayed in inspect tree when setting its DropDownStyle as DropDown or DropDownList.

### ISSUE THREAD

**Issue by Amy-Li03** [dotnet/winforms#3654]
* .NET Core Version: 
.NET SDK 5.0.100-rc.1.20377.3

* More Info:
This is not a regression issue, it also reproduce on .NET Core 3.1.

* Have you experienced this same bug with .NET Framework?: 
No

**Problem description:**
When setting ComboBox's DropDownStyle as DropDown or DropDownList, the list item of comboBox doesn't display below Form1 window node, it incorrectly displayed below Desktop1 panel node.
![Core](https://user-images.githubusercontent.com/62929087/88614276-fe2f4e00-d0c1-11ea-8f62-07c6a7702dcf.png)

**Expected behavior:**
The list items of comboBox's UIA hierarchy should display below Form1 window node.
![Framework](https://user-images.githubusercontent.com/62929087/88614281-012a3e80-d0c2-11ea-917b-cfaceaa212be.png)

**Minimal repro:**
1. Launch Inspect tool.
2. Create WinForms core project with a comboBox control.
3. Set the DropDownStyle property of comboBox as DropDown or DropDownList.
4. Run the project.
5. Focus on the list items of comboBox, then observe the UIA hierarchy in inspect tree.

**Comment by SergeySmirnov-Akvelon** [comment]
Hi @Amy-Li03. I cannot reproduce it on my local environment. ComboBoxes with "DropDown" or "DropDownList" styles are displayed correctly in Inspector. 
![Issue-3654-dropdown](https://user-images.githubusercontent.com/23376742/97010658-60c3a900-154e-11eb-9523-3546f0c36018.png)
![Issue-3654-dropdownlist](https://user-images.githubusercontent.com/23376742/97010681-6a4d1100-154e-11eb-8b90-07d703839be3.png)

Could you please clarify what actions you are taking to reproduce this? Also could you please attach the project where you reproduce this?

**Comment by Amy-Li03** [comment]
Hi @SergeySmirnov-Akvelon, we still can repro this issue on latest VS 16.9 P2 + .NET SDK 6.0, have uploaded the demo app and repro steps video files. Any other question please let me know.
[DemoApp.zip](https://github.com/dotnet/winforms/files/5442438/DemoApp.zip)
![reproSteps](https://user-images.githubusercontent.com/62929087/97247674-19356980-183b-11eb-9533-2870bfb28e2e.gif)

**Comment by SergeySmirnov-Akvelon** [comment]
Hi @Amy-Li03. It works correctly on my environment:
- Microsoft Windows Version 10.0.19041.388
- NET Core Version: .Net SDK 5.0.100-rc.2.20479.15, last master
- Vistual Studio Version 16.8.0 Preview 5.0
- Inspector Version 7.2.0.0 OLEACC Version 7.2.194.041.1 (WinBuild. 160101.0800)

![issue-3654](https://user-images.githubusercontent.com/23376742/97272303-bfd13880-1842-11eb-8843-9c5da8da2447.gif)

Could you please clarify your Windows and Inspector versions?

**Comment by Amy-Li03** [comment]
Hi @SergeySmirnov-Akvelon, my environment as following:

Microsoft Windows Version: 1909 (OS Build 18363.1139)
NET Core Version: .Net SDK 6.0.100-alpha.1.20516.5
Vistual Studio Version: 16.9.0 Preview 2.0 main branch latest build
Inspector Version:  7.2.0.0 OLEACC Version 7.2.18362.1 (WinBuild. 160101.0800)

In addition, have added you as administrator on my machine, IP have sent to you in teams.

**Comment by SergeySmirnov-Akvelon** [comment]
@Amy-Li03, thank you so much. I reproduced it using Inspect with your version

**Comment by John-Qiao** [comment]
Verified the issue with latest .NET SDK 6.0.100-alpha.1.21062.11, it has been fixed, please check below screenshot:
![3654-TestResult](https://user-images.githubusercontent.com/45864985/104411503-77096000-55a5-11eb-8091-bb1040fc6bf5.png)
